src/Template/Bake/Model/entity.ctp

```diff
@@ -20,6 +20,9 @@ use Cake\ORM\Entity;
 
 /**
  * <%= $name %> Entity.
+<% foreach ($fields as $field): %>
+ * @property $<%= $field %>
+<% endforeach; %>
```

Comment:
There are tests that check the generated files. Those will need to be updated. The travis.ci build has the details of which comparison files need changes.